custom/dhis2/tasks.py

```diff
@@ -288,64 +214,60 @@ def gen_children_only_ours(domain):
             yield CommCareCase.wrap(doc)
 
 
-@periodic_task(run_every=crontab(minute=4, hour=4))  # Run daily at 04h04
-def sync_child_entities():
+# TODO: Use case forwarding, or form forwarding of registration forms
+@periodic_task(run_every=timedelta(minutes=5))  # Run every 5 minutes to match forwarded forms
+def sync_cases():
```

Comment:
@kaapstorm this is clogging up the celery queues and making other parts of the site not work. I just disabled it in https://github.com/dimagi/commcare-hq/pull/5550. I suspect the slowness is at least partly to do with the fact that you are looping through every domain in the database every 5 minutes (I didn't look closely enough to see whether it's also the fact that it's doing lots of work for the dhis domains or whether this issue caused additional slowness https://github.com/dimagi/commcare-hq/pull/5551). is the TODO above something we can change soon? otherwise we may need to either drastically reduce the time this function takes by optimizing it better or reduce the frequency significantly.